File: backend/src/SecurityService_Core/Models/Enums/OrderProcessingAction.cs
namespace SecurityService_Core.Models.Enums
{
    /// <summary>
    /// Действие с заявкой
    /// </summary>
    public enum OrderProcessingAction
    {
        /// <summary>
        /// Отправить в обработку, изменив статус с 0 на 1
        /// </summary>
        Send = 0,

        /// <summary>
        /// Отменить заявку, сменив статус с 0 или 1 на -2
        /// </summary>
        Decline = 1,

        /// <summary>
        /// Дублировать заявку, сделав её копию под новым ID, но со статусом 0 (вновь заведённая)
        /// </summary>
        Double = 2
    }
}

File: frontend/src/app/enums/orderProcessingAction.ts
export enum OrderProcessingAction {
    /** Отправить в обработку, изменив статус с 0 на 1 */
    Send = 0,
    /** Отменить заявку, сменив статус с 0 или 1 на -2 */
    Decline = 1,
    /** Дублировать заявку, сделав её копию под новым ID, но со статусом 0 (вновь заведённая) */
    Double = 2
  }
  